iceberg/iceberg-common/build.gradle.kts

```diff
@@ -83,6 +105,7 @@ dependencies {
   annotationProcessor(libs.lombok)
   compileOnly(libs.lombok)
 
+  testImplementation(project(":server-common"))
```

Comment:
Why do we require "server-common" here?

Reply:
because IcebergConfig overwites the default `HTTP port`,  we add a test to check whether it works in `JettyServerConfig`,  like:
```java
  public void testIcebergHttpPort() {
    Map<String, String> properties = ImmutableMap.of();
    IcebergConfig icebergConfig = new IcebergConfig(properties);
    JettyServerConfig jettyServerConfig = JettyServerConfig.fromConfig(icebergConfig);
    Assertions.assertEquals(
        JettyServerConfig.DEFAULT_ICEBERG_REST_SERVICE_HTTP_PORT, jettyServerConfig.getHttpPort());
    Assertions.assertEquals(
        JettyServerConfig.DEFAULT_ICEBERG_REST_SERVICE_HTTPS_PORT,
        jettyServerConfig.getHttpsPort());
```